docs/conf.py

```diff
@@ -5,9 +5,9 @@
 import sys
 import sphinx_rtd_theme
 
-sys.path.insert(0, os.path.abspath("../microsoft"))
```

Comment:
This doesn't seem like a 1:1 mapping like all the other changes. I assume the path before was "/microsoft/featuremanagement". Wouldn't this change be making it "featuremanagement/featuremanagement"?

Reply:
? This is the file path we just removed `microsoft` from the file path, nothing else changed. This is just pointing to where our code is for autodoc generation.

Reply:
Changing this to 
```python
sys.path.insert(0, os.path.abspath("../"))
```

Fixes the sphinx build and I makes sense to me

Reply:
Solved with new install